coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/budget/core/ProposalBudgetViewHelperServiceImpl.java

```diff
@@ -205,4 +207,15 @@ public void setKcBusinessRulesEngine(KcBusinessRulesEngine kcBusinessRulesEngine
 		this.kcBusinessRulesEngine = kcBusinessRulesEngine;
 	}
     
+	public boolean isBudgetMarkedForSubmission() {
+		ProposalBudgetForm form = (ProposalBudgetForm)ViewLifecycle.getModel();
+		boolean budgetMarkedForSubmission = false;
+		Budget finalBudget = form.getDevelopmentProposal().getFinalBudget();
```

Comment:
@leoherbie yeah thought of it, but then it is just one condition that we check. Not sure if there is any advantage in moving it to a service